### ISSUE TITLE
Yield exemption flag per token

### ISSUE THREAD

**Issue by jubeira** [balancer/balancer-v3-monorepo#171]
Apperently there's a strong case to have this, besides avoiding double tax with nested pools (which was acceptable in principle).

There are some tokens that will have some form of rate provider, but are not yield bearing. E.g. some EUR or FX tokens will have their rate (vs USD) injected using an oracle.

This could be solved at pool level, but solving it at the Vault could simplify things down the line. Extract from @danielmkm in Slack:

> imo, live balances are meant to manage decimal and rate scaling at the vault level so that the pool does not need to worry about the extra complexity. We've previously discussed the duality that will exist between pairs like aDAI/DAI and wstETH/weth, where one pair can be unwrapped on the fly, while the other cannot. As both of these patterns will likely be very common, we've been operating under the assumption that native support for both varities should exist in the vault. Happy to revisit this, but I don't see a lot of value in having the restriction, described here, and it could potentially create quite some confusion as there would be a rate implementation at the vault that would only facilitate a very specific use case.

In other words, we're doing a lot of effort to have the right scaling at the Vault, and it'd seem reasonable to include this new use case as well.

This should be added after #100 is complete and merged (or at least under review).
